src/assets/styles/_base.scss

```diff
@@ -40,7 +41,7 @@ h6 {
 }
 
 a {
-  text-decoration: underline;
+  text-decoration: none;
```

Comment:
**Question**: Have you talked to design about whether the choice to not underline links can be reconsidered?

It’s a terrible practice and we really shouldn’t be doing it.

Reply:
I think we all talked to design a while ago. This was the team decision https://github.com/kumahq/kuma-gui/issues/738 as linked to in the description above.

There're follow up PRs to come that add slightly different treatment on hover and removing all the `text-decoration: none`s we have throughout the codebase, I wasn't sure whether to do that here or in subsequent PRs. I'm happy to do follows ups but lmk if you prefer otherwise.

Apart from that, AFAIK this is the decision as its stood for a while now and hasn't changed from what I understand. If you still disagree then feel free to bring it up in sync with the team.

> It’s a terrible practice and we really shouldn’t be doing it.

^ in your opinion?

Reply:
We made that decision as a team (which did not involve design) because we wanted to follow what other projects were doing, but we did note that this wasn’t ideal and that a conversation with design should take place to reconsider that.

> ^ in your opinion?

Not opinion but a matter of fact. Links need to be underlined as otherwise the sole distinguishing visual element is its color which isn’t enough for some visually-impaired people. Adding a differentiation on hover does absolutely nothing to help that as by the point you hover over something you already harbor a guess as it being interactive.

Reply:
This is how it is in Konnect. We are aiming for consistency rather than things being different due to differences of opinion. If Konnect want to change their opinion then we stick to that, and we change as/when/if they do. As of right now this is how it is.

Reply:
Are we good to resolve this one?

Reply:
Yes, but I had hoped for this conversation to be brought to design which is actually what we agreed as a team.

Reply:
Awesome. If you still feel strongly about this feel free to take that to design or bring it with you to GUI or larger team sync and I personally would be happy to chat further. In the meantime https://github.com/kumahq/kuma-gui/issues/738 is what has been agreed as a team. If Konnect changes look and feel as a result of that, then we can too.